app/src/main/java/com/flowfoundation/wallet/EnvConfigModule.java

```diff
@@ -0,0 +1,227 @@
+package com.flowfoundation.wallet;
```

Comment:
This project is based on kotlin, could you submit kotlin file instead of java file ?